test/fixtures/data.py

```diff
@@ -144,6 +144,38 @@ def revolution_inc(
     }
 
 
+@pytest.fixture(scope="session")
+def winner_inc(
+    crn_companies: DataFrame, duns_companies: DataFrame, cdms_companies: DataFrame
+) -> dict[str, str]:
+    """
+    Winner Inc. as it exists across all three datasets.
+
+    UUIDs are converted to strings to mirror how Matchbox stores them.
+
+    Based on the above fixtures, should return:
+
+    * Three CRNs
+    * Zero DUNS
+    * Two CDMS
+    """
```

Comment:
could you... assert this is true? Not sure how I feel abbout assertions in a fixture, but it would increase my confidence in the stability of tests without having to just trust that the docstring is true now and in the future